austation/code/modules/hydroponics/grown/gympie_gympie.dm

```diff
@@ -62,3 +64,15 @@
 					G.visible_message("<span class='notice'>The Gympie Gympie violently shakes its leafs at you!</span>")
 					qdel(src)
 
+/obj/item/seeds/gympie_gympie/attackby(obj/item/W, mob/user)//Name the child
+	if(istype(W, /obj/item/pen))
+		var/new_name = stripped_input(user, "What would you like the name to be?")
+		if(!user.canUseTopic(src, BE_CLOSE))
+			return
+		if(new_name)
+			babys_first_name = new_name
+			name = new_name
+		return
+	else
+		return ..()
```

Comment:
this else is redundant, since you already return anyway if the if block triggers
```suggestion
	return ..()
```